mass_editing/__openerp__.py

```diff
@@ -21,10 +21,11 @@
 ##############################################################################
 {
     "name": "Mass Editing",
-    "version": "8.0.1.3.0",
+    "version": "9.0.1.1.0",
     "author": "Serpent Consulting Services,Odoo Community Association (OCA)",
     "contributors": [
         "Oihane Crucelaegui <oihanecrucelaegi@gmail.com>",
+        "Florent Thomas <contact@mind-and-go.com>"
```

Comment:
Please create a proper README file following https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md and https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
Remove description key